restorer.go

```diff
@@ -44,7 +45,8 @@ func (res *Restorer) restoreTo(dst string, dir string, treeID backend.ID) error
 	}
 
 	for _, node := range tree.Nodes {
-		if err := res.restoreNodeTo(node, dir, dst); err != nil {
+		excluded, err := res.restoreNodeTo(node, dir, dst)
```

Comment:
I think maybe this is where the filtering should happen and then `continue` the loop before even calling `restoreNodeTo`. Thoughts?